zero-noexert-exhaustion bugfix script, doc, and control-panel registry entry (#1053)
DFHack/scripts: docs/fix/noexert-exhaustion.rst

> fix/noexert-exhaustion
> ===========================

Reviewer: ```suggestion
fix/noexert-exhaustion
======================
```

.. dfhack-tool::

>     :summary: Sets Exhaustion of NOEXERT units to 0, curing 'Tired'

Reviewer: ```suggestion
    :summary: Prevents units with the NOEXERT tag from getting tired when training.
```

Reviewer: 56 chars, but that is close enough

    :tags: fort bugfix units

> Zeroes the Exhaustion counter of all NOEXERT units (including Vampires, Necromacers, and Intelligent Undead),

Reviewer: ```suggestion
This tool zeroes the "exhaustion" counter of all NOEXERT units (e.g. Vampires, Necromancers, and Intelligent Undead),
```

fixing any that are stuck 'Tired' from an activity that doesn't respect NOEXERT. This is not a permanent fix -
the issue will reoccur next time they partake in an activity that does not respect the NOEXERT tag.

Running this regularly works around :bug:`8389`, which permanently debuffs NOEXERT units and prevents them from
properly partaking in military training. It should be run if you notice Vampires, Necromancers, or Intelligent

> Undead become 'Tired'. Enabling this script via `control-panel` or `gui/control-panel` will run it often enough to

Reviewer: ```suggestion
Undead becoming 'Tired'. Enabling this script via `control-panel` or `gui/control-panel` will run it often enough to
```
